README.md

```diff
@@ -8,11 +19,31 @@ Eloquent allows you to read and browse many different bibles translations in dif
 
 And because Eloquent is based on the [Sword project](https://crosswire.org/sword/index.jsp), it is compatible with an uncountable number of texts that are available via the built-in module manager in over 50 languages.
 
+Key Features:
+
+Multi-translation Support: View and compare different Bible translations side by side.
+Search Capabilities: Easily search through different texts for specific verses or keywords.
+Module Manager: Install and manage Bible translations, devotionals, and other resources directly from the app.
+Commentary and Lexicon Support: Gain deeper understanding with integrated commentaries and lexicons.
+Cross-app Integration: Access the Bible in any program using macOS services.
+
+Usage Instructions:
+
+Reading the Bible: Open the app, and choose your preferred translation from the module manager.
+Opening Multiple Pages: You can open multiple Bible translations or commentaries by selecting the "New Tab" option. Each tab can display a different resource side by side.
+Text Search: Use the search bar at the top to search for specific verses, keywords, or phrases within your selected translations.
 
 # Download releases
 Releases can be downloaded under this link:
 https://github.com/mdbergmann/Eloquent/releases
 
+Installation Instructions
+macOS Installation
+Download Eloquent: Go to the Releases page and download the latest version for macOS.
+Install Dependencies: Ensure that you have all necessary libraries installed, such as Sword libraries. Use Homebrew for easy installation:
+bash
```

Comment:
Bash and pre-installed sword is usually not needed. Eloquent comes with sword included.